game-core/src/main/java/games/strategy/triplea/delegate/BattleDelegate.java

```diff
@@ -834,8 +833,7 @@ private void doScrambling() {
         if (scramblers.isEmpty()) {
           continue;
         }
-        final Map<Territory, Collection<Unit>> toScramble =
-            getRemotePlayer(defender).scrambleUnitsQuery(to, scramblers);
+        final var toScramble = getRemotePlayer(defender).scrambleUnitsQuery(to, scramblers);
```

Comment:
There is a cost to losing type info, I'm not sure if the code neatness is worth the loss of the type information. Perhaps if the variable were named: `unitsToScrambleByTerritory`, then the type information would be more redundant. Any thoughts?

Reply:
In this case I'm fine reverting this change.
I had this stash laying around somewhere and I remeber stopping because at some point the changes were no longer good.